tests/unit/mocks/array_mock.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2023, Denys Denysyev (ddenysyev@purestorage.com)
+# GNU General Public License v3.0+ (see COPYING.GPLv3 or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+class MockArray:
+    def __init__(self, params) -> None:
```

Comment:
Inconsistent type hinting, the commit removes type hint in _get_verbosity but adds it here